lib/src/uptech_growthbook_wrapper.dart

```diff
@@ -63,6 +63,15 @@ class UptechGrowthBookWrapper {
     return await _client.refresh();
   }
 
+  Future<void> setAttributes(Map<String, dynamic> attributes) async {
+    if (_client.context.attributes != null) {
+      _client.context.attributes?.addEntries(attributes.entries);
+    } else {
+      _client.context.attributes = attributes;
+    }
+    refresh();
```

Comment:
Why call `refresh()` here? I thought `refresh()` just caused it to go fetch the state from the server. If that is all it does I am not sure it makes sense to couple the refresh to the updating of attributes.

I am also guessing this is the only reason why this is an asynchronous call vs a synchronous call. We should always opt for synchronous calls whenever possible.